assets/js/base/components/product-list/style.scss

```diff
@@ -61,9 +63,15 @@
 	}
 }
 
+.edit-post-visual-editor .editor-block-list__block .wc-block-grid__product-title,
+.editor-styles-wrapper .wc-block-grid__product-title,
```

Comment:
I was taking a look at this, and these two selectors above don't seem to make a difference for me. Are they required for a specific theme or could we remove them?

Reply:
Storefront - it adds it's own styles which take priority.